python/cudf/cudf/_lib/rolling.pyx

```diff
@@ -72,8 +73,13 @@ def rolling(inp, window, min_periods, center, op):
             fill_value = inp.default_na_value()
             mask = cudautils.make_empty_mask(null_count)
         data = cudautils.full(
-            inp.data.mem.size, fill_value, inp.data.mem.dtype
+            inp._data_view.size, fill_value, inp._data_view().dtype
```

Comment:
What's the difference between `._data_view` and `._data_view()` here? Also not crazy about those names for viewing the Buffer as a numba device array.

Reply:
Fixed